_posts/2017/2017-03-21-continuous-documentation-with-ansible.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: post
+title: "Continuous Documentation with Ansible"
+date: "2017-03-21 06:39:20"
+image: blog-header/scrum-board.jpg
```

Comment:
Where is this image from?
Please use one from these portals: 
https://pixabay.com/de/
https://www.pexels.com/
https://unsplash.com/
as we don't have issues with the image rights there. Thanks